{"Modified":"2022-08-05T17:44:00","Published":"2021-03-26T17:15:00","access":{"authentication":"NONE","complexity":"MEDIUM","vector":"NETWORK"},"assigner":"secalert@redhat.com","capec":[],"cvss":8.3,"cvss-time":"2022-08-05T17:44:00","cvss-vector":"AV:N/AC:M/Au:N/C:P/I:P/A:C","cwe":"CWE-787","id":"CVE-2021-20285","impact":{"availability":"COMPLETE","confidentiality":"PARTIAL","integrity":"PARTIAL"},"last-modified":"2022-08-05T17:44:00","references":["https://github.com/upx/upx/issues/421","https://bugzilla.redhat.com/show_bug.cgi?id=1937787"],"summary":"A flaw was found in upx canPack in p_lx_elf.cpp in UPX 3.96. This flaw allows attackers to cause a denial of service (SEGV or buffer overflow and application crash) or possibly have unspecified other impacts via a crafted ELF. The highest threat from this vulnerability is to system availability.","vulnerable_configuration":[{"id":"cpe:2.3:a:upx_project:upx:3.96:*:*:*:*:*:*:*","title":"cpe:2.3:a:upx_project:upx:3.96:*:*:*:*:*:*:*"}],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":["cpe:2.3:a:upx_project:upx:3.96:*:*:*:*:*:*:*"]}